features/openhab-addons-external/pom.xml

```diff
@@ -120,6 +120,7 @@
                                 <artifact><file>src/main/resources/conf/sapp.cfg</file><type>cfg</type><classifier>sapp</classifier></artifact>
                                 <artifact><file>src/main/resources/conf/satel.cfg</file><type>cfg</type><classifier>satel</classifier></artifact>
                                 <artifact><file>src/main/resources/conf/snmp.cfg</file><type>cfg</type><classifier>snmp</classifier></artifact>
+                                <artifact><file>src/main/resources/conf/sonance.cfg</file><type>cfg</type><classifier>snmp</classifier></artifact>
```

Comment:
Build failed on this line.

Please update to `<classifier>sonance</classifer>`.